docs/light-client/README.md

```diff
@@ -66,39 +66,40 @@ Once IRISLCD is started, you can open `localhost:1317/swagger-ui/` in your explo
     7. `GET /auth/accounts/{address}`: Get the account information on blockchain
```

Comment:
/auth/accounts/{address} --> /bank/accounts/{address}

Reply:
The URL is /auth/accounts/{address}  in the code, need to be modified?

Reply:
Yes, please do.